package.json

```diff
@@ -33,70 +33,67 @@
 		"compodoc": "compodoc -p ./tsconfig.compodoc-json.json -e json -d .storybook",
 		"build-compodoc": "compodoc -p ./tsconfig.compodoc-html.json -d compodoc-static",
 		"build-storybook": "ng run storybook:build",
-		"lint": "ng lint"
+		"lint": "ng lint",
+		"lint:scss": "stylelint --config .stylelintrc.json packages/scss/src/**/*.scss"
```

Comment:
It this called anywhere? It could replace the gulp command

Reply:
no it's not called anywhere. I dropped `gulp-styleint` in favor of this npm script in order to drop some unmaintained dependencies